app/src/main/java/com/infomaniak/mail/ui/login/IntroFragment.kt

```diff
@@ -66,8 +64,7 @@ class IntroFragment : Fragment() {
         when (navigationArgs.position) {
             0 -> {
                 pinkBlueSwitch.isVisible = true
-                val tabIndex = introViewModel.currentAccentColor.value?.introTabIndex
-                    ?: UiSettings.DEFAULT_ACCENT_COLOR.introTabIndex
+                val tabIndex = loginViewModel.currentAccentColor.value?.introTabIndex!!
```

Comment:
Is it dangerous to !! a LiveData ?

Reply:
If there is a possibility that the `livedata` becomes null yes

Reply:
But we can make it safer, we should use a `let` here, 
and move the default value from mainActivity into the `ViewModel` directly.